test/spoom/cli/srb/tc_test.rb

```diff
@@ -346,8 +346,8 @@ def test_output_errors_to_junit_xml
                   <![CDATA[In file errors/errors.rb:
                 10 |b = Foo.new(42)
                                 ^^
-                <GIT_LINK>/rbi/core/basic_object.rbi#L228: `initialize` defined here
-                 228 |  def initialize(); end
+                <GIT_LINK>/rbi/core/basic_object.rbi#L230: `initialize` defined here
```

Comment:
Needing to update line numbers when RBIs are updated is pretty crappy, but switching to a pattern would be a fair bit of effort.